stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,72 @@
+{{- $svcname := "weave-scope" -}}
+{{- $svcport := 80 -}}
+
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: {{ .Values.serviceAccount | default "weave-scope" }}-agent
+    {{- include "helm_std_labels" . | indent 4 }}
+    component: agent
+  name: {{ template "fullname" . }}-agent
+  annotations:
+    {{- include "scope_annotations" . | indent 4 }}
+spec:
+  selector: 
+    matchLabels:
+      name: {{ template "fullname" . }}-agent
+      app: {{ template "fullname" . }}
```

Comment:
I don't have a template called "name" I don't think, I'll add that one in from the nginx example and make this and the other corresponding changes.